test/conftest.py

```diff
@@ -57,3 +51,22 @@ def git_access_mock():
     git_access_mock.get_files_of_commit.return_value = ["src/udfclient.cpp"]
     git_access_mock.get_last_commit_message.return_value = "last commit"
     return git_access_mock
+
+
+@dataclass
+class DockerConfig:
+    docker_user: str
+    docker_password: str
+    docker_build_repository: str
+    docker_release_repository: str
```

Comment:
You may remove the prefixes of the members. I think it should be clear from the `context` or `type`

```python
config = DockerConfig(...)
config.username 

# alternatively
docker_config = DockerConfig(...)
docker_config.username 
```

```suggestion
class DockerConfig:
    username: str
    password: str
    build_repository: str
    release_repository: str
```

Reply:
I think this file can actually be removed, need to check if it still is used

Reply:
it doesn't get used so I removed it